feat: Mark members with cards on table

bombsimon opened the pull request:
This PR will add support to show an indicator next to the members who currently have cards on the table.

---
Thanks for this great tool, really appreciated!

I've noticed when the number of participants increases it's getting harder to see when everyone have put their card on the table. This will not change the possibility to sit out, it will just make it easier to spot who have put a card on the table.

<img width="605" alt="image" src="https://user-images.githubusercontent.com/3248948/216478513-20285f4a-4a46-467d-8c79-770eb1a0542b.png">

pedsm commented:
Sorry this took me ages to get to, that said this looks really cool I needed to sort some of the build process for this repo to move it into Turbo repo but now that it's done I will rebase this one and get it in once I have some time 🚀

pedsm commented:
@bombsimon I can't do the rebase on the fork but I opened #48 which is a copy of this PR like for like. I can merge that and close this but if you would like the contributor credit let me know and you can rebase this. The changes should work but I have moved the `/frontend` dir into `/apps/frontend`

bombsimon committed:
feat: Mark members with cards on table

This PR will add support to show an indicator next to the members who
currently have cards on the table.

bombsimon commented:
Thanks! I know how life is so don't worry about not getting to this sooner. 😄 

I don't mind but since you already did the job it was easy enough for me to rebase and push on this branch. Feel free to merge this or your branch, whichever you prefer. They should be the same now anyways!

Thanks again for this tool, it's great!
